static/js/components/VideoPlayer.js

```diff
@@ -109,8 +110,8 @@ export default class VideoPlayer extends React.Component<*, void> {
             tracks[idx].addEventListener("modechange", function() {
               sendGAEvent(
                 "video",
-                "modechange",
-                `Subtitles for ${this.label} ${this.mode}`,
+                `Subtitles ${this.label} ${this.mode}`,
```

Comment:
Is it intentional to swap these two arguments? What do `action` and `label` mean for google analytics?

Reply:
Intentional, now the label should always be equal to `video.key` for all video events including this one, and the `Subtitles...` string should be the event.